src/utils/gnarrc/index.ts

```diff
@@ -2,15 +2,12 @@ import { writeFileSync } from 'node:fs'
 import axios from 'axios'
 import { stdOut } from '../std-out'
 
-const API = axios.create({
-  baseURL: 'https://raw.githubusercontent.com/TheGnarCo/.gnarrc/main/',
-})
+const baseURL =
+  process.env?.DEV_URL || 'https://raw.githubusercontent.com/TheGnarCo/.gnarrc/prod-cli/'
 
-export class Gnarrc {
-  static API = axios.create({
-    baseURL: 'https://raw.githubusercontent.com/TheGnarCo/.gnarrc/main/',
-  })
```

Comment:
Was this variable even being used before?

Reply:
nope! it was redundant.